### ISSUE TITLE
{Linked:Bug3118618} [Adaptive Cards > Action.Toggle Visibility] - Screen Reader is not conveying the 'Expanded/Collapsed' state for the more option button. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8654]
### Target Platforms

Other

### SDK Version

1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/explorer/Action.ToggleVisibility.html
OS Version: 23H2(OS Build 25915.1000)
Browser Version: Edge Dev 117.0.2033.0
Screen Reader: Narrator

**Repro Steps:**
1. Open the URL on Edge Dev browser.
2. Press tab key and navigate to the 'More Option' button under 'Please confirm your data'.
3. Press enter key and observe the issue.

**Actual Result:**
While navigating to the 'More Option' button on pressing enter key, screen reader is not conveying the state information as 'Expanded'. It is simply announcing the information about the items present under it.

**Expected Result:**
Screen reader should convey the state information when user activates the 'More Option' button. It should announce as **'More Option button expanded then it should announce the items present under it.'**

**User Impact:**
Users who rely on screen readers will be affected if screen reader does not announce expand/collapsed state info for the controls present in the Action.toggle visibility screen. Due to this user will get confused and will not get idea that control is getting expanded/collapsed and this may cause distortion.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B507B5D90-F1B9-4AE1-91D1-2F1F2D6C7C1D%7D&file=MAS%204.1.2%20%E2%80%93%20Name%2C%20Role%2C%20Value.docx&action=default&mobileredirect=true&share=IQGQXXtQufHhSpHRLx8tbHwdAUoz-PLo6lZ2eagKqmDeOGc

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fforms.office.com%2fpages%2fresponsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-ssachan%40microsoft.com%7c68d0243104da4caeb08708d9ba4d6c6b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637745663988426007%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000&sdata=YMsEOknKWOki3Fi%2BatsKabxS%2BM1tfTrgnws52jzpI7M%3D&reserved=0).

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/95913748/6ae158f0-a128-454c-b0e1-607665c50439

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This bug can be originally tracked using the URL below.

https://dev.azure.com/domoreexp/MSTeams/_workitems/edit/3118618


**Snippet:**

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/f6be0e70-b4e0-4881-bbfd-b8254d1568fa)

**Comment by BhargaviAnnadevara-MSFT** [comment]
Any update on this issue?

**Comment by vagpt** [comment]
This issue is working fine on the environment below, hence closing this issue.

**Test Environment:**
URL: https://adaptivecards.io/explorer/Action.ToggleVisibility.html
OS Version: 23H2(OS Build 25977.1000)
Browser Version: Version 120.0.2186.2 (Official build) dev (64-bit)
Screen Reader: Narrator
